core/scss/components/activity-button/activity-button.scss

```diff
@@ -46,6 +46,25 @@
       }
     }
 
+    &__contact-card,
+    .#{$dark-class} & {
+      @include button-color(
+        $color: $button__font-color--alt,
+        $bg: $button--mint__background-color,
```

Comment:
this color doesn't match the spec. Need new color map for the correct color #8AC6C9
@pauljeter